plans/dev/main.fmf

```diff
@@ -0,0 +1,28 @@
+/:
+  inherit: false
+
+execute:
+  how: tmt
+discover:
+  how: fmf
+  filter: "tag: dev"
+
+prepare:
+  - name: Install common base packages
+    how: install
+    package:
+      - python3-pip
+      - git
+
+/with-tmt:
```

Comment:
The name does not say much. If we go with `source`, this one could perhaps be `/plans/source/sanity`?

Reply:
Yeah, the naming is tricky. I would want to keep the `with`/`without` tmt suffixes, because those should be the only distinction between the tests run there. Maybe `/plans/sanity` has the closest naming, but that onle still misses the fact that these are distro-agnostic plans.